modules/ROOT/pages/_partials/data-sync/server-using-mutations-with-a-data-sync-server-and-a-data-sync-client.adoc

```diff
@@ -0,0 +1,122 @@
+[id="server-using-mutations-with-a-data-sync-server-and-a-data-sync-client-{context}"]
+= Using Mutations with a Data Sync Server
+
+Build a Data Sync Server and query the mutation.
+The Data Sync Server will be running locally in this procedure.
+
+.Prerequisites
+
+* link:https://github.com/jstaffor/mobile-docs/blob/master/modules/ROOT/pages/_partials/sync/server-getting-started.adoc[Getting Started with Hello World Data Sync].
+* You are familiar with {sync-server} operations, for example, stopping and starting {sync-server}.
+
+.Procedure
+
+. If the {sync-server} is running, stop it.
++
+. Edit the `index.js` file and replace its contents with the following content:
```

Comment:
Have noticed that 'index.js' is used throughout the doc but 'index-1.js' is the file that is being created at the start of the doc, in the "Getting Started with Hello World Data Sync" section.

Reply:
I think this is an aspect for the next phase of this work where we consolidate the two streams of work and ensure there is consistency from the beginning to the end of the user story.